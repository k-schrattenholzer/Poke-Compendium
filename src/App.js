import './App.css';
import Compendium from './views/Compendium/Compendium.jsx';

function App() {
  return (
    <div className="App">
      <Compendium />
      <p>its a compendium!</p>
    </div>
  );
}

export default App;
