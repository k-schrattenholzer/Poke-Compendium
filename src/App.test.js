import { render, screen } from '@testing-library/react';
import App from './App';

it('renders compendium', () => {
  render(<App />);
  const linkElement = screen.getByText(/compendium/);
  expect(linkElement).toMatchSnapshot();
});
