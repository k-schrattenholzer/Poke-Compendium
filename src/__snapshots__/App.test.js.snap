// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders compendium 1`] = `
<p>
  its a compendium!
</p>
`;
